src/evaluator.rs

```diff
@@ -8,18 +11,23 @@ use crate::parser::{ClearText, Encoding, ParsedEncodedWord, ParsedEncodedWords};
 #[derive(Error, Debug)]
 pub enum Error {
     #[error(transparent)]
-    DecodeUtf8Error(#[from] string::FromUtf8Error),
+    DecodeUtf8(#[from] string::FromUtf8Error),
```

Comment:
Why did you rename error variants?

- it is not part of the scope of bumping base64 crate
- it does not follow the [convention](https://rust-lang.github.io/api-guidelines/naming.html#names-use-a-consistent-word-order-c-word-order) initially set up
- those changes can be technically considered breaking since it is a lib